prompts/agent/container_tool.txt

```diff
@@ -0,0 +1,40 @@
+To help write a simple but valid fuzz target, your can investigate any files in the fuzz target's build environment by iteratively send me a BASH command, and I will provide you with the output of the command.
```

Comment:
should we rename this directory "agent_tools" to be clearer?

Reply:
Fixed.
I was thinking of putting everything into `prompts/agent/` for now and replacing existing prompts with them once this is ready, but your suggestion is better.
I moved it to `prompts/tool/` now.